posthog/api/test/__snapshots__/test_action_people.ambr

```diff
@@ -31,7 +31,7 @@
         HAVING max(is_deleted) = 0) person ON person.id = pdi.person_id
      WHERE team_id = 2
        AND event = '$pageview'
-       AND toStartOfDay(toDateTime(timestamp, 'UTC')) >= toStartOfDay(toDateTime('2020-01-08 00:00:00', 'UTC'))
+       AND timestamp >= toStartOfDay(toDateTime('2020-01-08 00:00:00'), 'UTC')
```

Comment:
I'm confused by this difference. Just sanity checking: If a user is in a none utc timezone, will this comparison hold up? It would be comparing a UTC timestamp to a non utc timestamp

Reply:
toStartOfDay(time, timezone) returns the start of the day in the target timezone in utc. So if the timezone was `US/Pacific` it would return 2020-01-08 08:00:00 as that's the start of the day in us/pacific

Reply:
I mean what happens if the statement reads `timestamp >= toStartOfDay(toDateTime('2020-01-08 00:00:00'), 'Pacific')`

Reply:
In that case it'll be `timestamp >= 2020-01-08 08:00:00`